cmd/terramate/cli/cli.go

```diff
@@ -187,8 +187,9 @@ type cliSpec struct {
 
 	Cloud struct {
 		Login struct {
-			Google bool `optional:"true" help:"authenticate with google credentials"`
-			Github bool `optional:"true" help:"authenticate with github credentials"`
+			Google bool `optional:"true" xor:"login_provider" help:"authenticate with google credentials"`
+			Github bool `optional:"true" xor:"login_provider" help:"authenticate with github credentials"`
+			SSO    bool `optional:"true" xor:"login_provider" help:"authenticate with SSO credentials"`
```

Comment:
Do you need to provide a flag? As the endpoint call latency is low and guess it based on the response. It is more convenient to end user as the organization name is now mandatory to sign with TMC

Reply:
Good question. Theoretically we can detect that current project uses SSO and do it automatically but I don't think this is a good idea. The `--google` is the default option and I think it's confusing if this is overridden if executed from a SSO based project.

@mariux wdyt?